README.md

```diff
@@ -143,6 +143,19 @@ export const config = {
 
 This exposes the MCP endpoint at `/tidewave/mcp`.
 
+**Logging** (optional): To capture application logs for debugging via the
+`get_logs` MCP tool, create an `instrumentation.ts` file:
+
+```typescript
+// instrumentation.ts
+export async function register() {
+  if (process.env.NEXT_RUNTIME !== 'edge') {
+    const { registerTidewaveLogger } = await import('tidewave/next-js/logger');
```

Comment:
Should we be worried about cases where we are in prod and the runtime is using node.js? Should we also check the environment?